### PR TITLE
Amam/themefy

### PR DESCRIPTION
**Pull request by cakasuma**

### CODE REVIEW

File: src/sass/app_2.scss [hunk 1]
```diff
@@ -1,6 +1,7 @@
 // Base
 @import 'app_2/_common/base/reset';
 @import 'app_2/_common/base/constants';
+@import 'app_2/_common/base/themes';
```
Comment (negarn) [inline src/sass/app_2.scss]: can you make this list alphabetic?

Comment (cakasuma) [reply src/sass/app_2.scss]: sure 👍

File: src/sass/app_2/_common/base/constants.scss [hunk 2]
```diff
@@ -10,24 +10,30 @@ $COLOR_PURPLE     : #4f60ae;
 $COLOR_RED        : #e31c4b;
 $COLOR_SKY_BLUE   : #2196f3;
 $COLOR_WHITE      : #ffffff;
+$COLOR_BLACK      : #000000;
```
Comment (negarn) [inline src/sass/app_2/_common/base/constants.scss]: this also should be on line 6 to be alphabetical

File: src/sass/app_2/_common/base/constants.scss [hunk 3]
```diff
@@ -10,24 +10,30 @@ $COLOR_PURPLE     : #4f60ae;
...
+$COLOR_DARK_BLUE_2         : #101320;
+$COLOR_DARK_BLUE_3         : #191c31;
+$COLOR_DARK_BLUE_4         : #202641;
+$COLOR_DARK_BLUE_5         : #2a3052;
+$COLOR_DARK_BLUE_6         : #555975;
+$COLOR_DARK_BLUE_7         : #7f8397;
+$COLOR_DARK_BLUE_8         : rgba(127, 131, 151, 0.3);
+$COLOR_DARK_GREEN_GRADIENT : linear-gradient(to top, rgba(16, 19, 32, 0), rgba(0, 148, 117, 0.16));
+$COLOR_DARK_RED_GRADIENT   : linear-gradient(to top, rgba(16, 19, 32, 0), rgba(227, 28, 75, 0.16));
+$COLOR_DARK_BLACK_GRADIENT : linear-gradient(to right, rgba(16, 19, 32, 1), rgba(16, 19, 32, 0));
```
Comment (negarn) [inline src/sass/app_2/_common/base/constants.scss]: should be on line 26, or if you want to keep all gradients together then on line 34

File: src/sass/app_2/_common/base/themes.scss [hunk 4]
```diff
@@ -0,0 +1,50 @@
...
+        text_active_color         : $COLOR_WHITE,
+        text_color                : $COLOR_BLACK,
+        chart_gradient_color      : $COLOR_LIGHT_WHITE_GRADIENT,
+        won_gradient_color        : $COLOR_LIGHT_GREEN_GRADIENT,
+        lost_gradient_color       : $COLOR_LIGHT_RED_GRADIENT,
+        navbar_background_color   : $COLOR_WHITE,
+        navbar_active_color       : $COLOR_LIGHT_BLACK_1,
+        disabled_color            : $COLOR_LIGHT_BLACK_3,
+        container_color           : $COLOR_WHITE,
+        background_button_color   : $COLOR_LIGHT_GRAY_2,
```
Comment (negarn) [inline src/sass/app_2/_common/base/themes.scss]: can you make these alphabetical too?

File: src/sass/app_2.scss (hunk 1 above)
Comment (negarn) [reply src/sass/app_2.scss]: oh, sure, leave it then

**Pull request merged**